Droit de la personne (#1)
Hornan7/Cardano-Constitution: README.md

(**TODO** Description and justification of this Community rule)

> ## Article III: The ADA Holders Rights
> 
>  1. Every person is entitled to fundamental freedoms such as freedom of conscience, freedom of religion, freedom of opinion, freedom of expression and freedom of association. And no Cardano Blockchain Protocol or Governance Mechanisms will prevail over them of these rights.
>  2. Everyone has the right to privacy of their wallet, and no Cardano protocol shall require its owner to provide identity information when creating key pairs or an address from a verification key. However, any Centralized entity, or Decentralized Autonomus Organization using the Cardano blockchain functionalities may require identity information for the use of their services.
>  5. Every person has the right to the recognition and exercise, in full equality, of their right to vote and to submit governance actions, without distinction, exclusion or preference based on race, colour, sex, gender identity or expression, pregnancy, sexual orientation, civil status, age except to the extent provided for by law, religion, political beliefs, language, ethnic or national origin, social condition, disability or the use of a means to compensate for that disability.Discrimination occurs when such a distinction, exclusion or preference has the effect of destroying or compromising that right.
>  9. No one may, in a legal act, stipulate a clause involving discrimination. Such a clause shall have no effect.
> 10. Everyone has the right to information, to the extent provided by the Constitution and the Cardano Blockchain must remain public and open-source.
> 12. Human rights and freedoms are exercised with respect for democratic values
> 14. 
> 

Author: ```suggestion
## Article III: The ADA Holders Rights
1. Every person is entitled to fundamental freedoms such as freedom of conscience, freedom of religion, freedom of opinion, freedom of expression and freedom of association. And no Cardano Blockchain Protocol or Governance Mechanisms will prevail over them of these rights.
2. Everyone has the right to privacy of their wallet, and no Cardano protocol shall require its owner to provide identity information when creating key pairs or an address from a verification key. However, any Centralized entity, or Decentralized Autonomus Organization using the Cardano blockchain functionalities may require identity information for the use of their services.
3. Every person has the right to the recognition and exercise, in full equality, of their right to vote and to submit governance actions, without distinction, exclusion or preference based on race, colour, sex, gender identity or expression, pregnancy, sexual orientation, civil status, age except to the extent provided for by law, religion, political beliefs, language, ethnic or national origin, social condition, disability or the use of a means to compensate for that disability.Discrimination occurs when such a distinction, exclusion or preference has the effect of destroying or compromising that right.
4. No one may, in a legal act, stipulate a clause involving discrimination. Such a clause shall have no effect.
5. Everyone has the right to information, to the extent provided by the Constitution and the Cardano Blockchain must remain public and open-source.
6. Community rights and freedoms are exercised with respect for democratic values.

```

(**TODO** Description and justification of this Community rule)

> ### Section 3: Community Bill of Rights

Reviewer: Cela pourrais remplacer l'article 3 qu'en pense tu? de cette facon j'ajouterais tout ce qui vien avec le `governance mechanism` de l'article 2 comme le 1 ADA = 1 Vote et tout le reste.

```suggestion
## Article III: The ADA Holders Rights
```

(**TODO** Description and justification of this Community rule)

### Section 3: Community Bill of Rights

>  1. Every person is entitled to fundamental freedoms such as freedom of conscience, freedom of religion, freedom of opinion, freedom of expression, freedom of peaceful assembly and freedom of association.

Author: allo

 2. Everyone has the right to privacy of their wallet, and no Cardano protocol shall require its owner to provide identity information when creating key pairs or an address from a verification key. However, any Centralized entity, or Decentralized Autonomus Organization using the Cardano blockchain functionalities may require identity information for the use of their services.
 4. Human rights and freedoms are exercised with respect for democratic values.
 5. Every person has the right to the recognition and exercise, in full equality, of the rights and freedoms of the person, without distinction, exclusion or preference based on race, colour, sex, gender identity or expression, pregnancy, sexual orientation, civil status, age except to the extent provided for by law, religion, political beliefs, language, ethnic or national origin, social condition, disability or the use of a means to compensate for that disability.Discrimination occurs when such a distinction, exclusion or preference has the effect of destroying or compromising that right.
 6. No person shall harass a person on one of the grounds referred to in Article 5.

>  7. No person may disseminate, publish or display in public a notice, symbol or sign involving discrimination or give authorization to that effect.

Reviewer: ```suggestion
```
ceci ne peux pas etre controler par quoi que ce soit dans la blockchain actuellement. La liberté d'expression vien avec ses risques. je pense par exemple a un NFT, par contre le metadata n'etant pas l'image en soit, cela peux probablement etre jeter dans la cours du IPFS. Et meme encore la c'est un terrain glissant qui se joue du coté des liberté d'expressions. la blockchain actuel n'empeche rien de tout cela, c'est a la communauté de le boycotté a mon avis. bref je prefere l'enlevé. si tu est daccord.

>  8. No person may, by discrimination, refuse to enter into a legal act concerning goods or services ordinarily offered to the public.

Reviewer: ```suggestion
```
Offtopic, ne concerne pas la gouvernance ni la blockchain. Et nous ne devrions pas empecher les utilisateur de Smart contract d'etre discriminatoire (si il le peuvent) dans la conception de leur code ou service centralisé) quoi que ce serait tres stupid de leur pars et tres peu vendeur.
